test/unit/currency-controller-test.js

```diff
@@ -0,0 +1,87 @@
+// polyfill fetch
+global.fetch = global.fetch || require('isomorphic-fetch')
```

Comment:
I [just asked our friend Jason](https://github.com/MetaMask/provider-engine/pull/117) if `isomorphic-fetch` works in react-native.  If it does, after that provider-engine bump, if we move to this everywhere, we'll be largely mobile-ready...